example/Gulpfile.js

```diff
@@ -79,7 +79,9 @@ toolkit.extendConfig({
 		],
 	},
 	css: {
-		baseFontSize: 16, // Used by postcss-pxtorem.
+		basefontsize: 16, // Used by postcss-pxtorem.
+        remreplace: false, // Used by postcss-pxtorem.
```

Comment:
This line and the following line use spaces for indentation. Rest of the file uses tabs.